### ISSUE TITLE
the problem of using label_smooth_eps=0.1 for classify

### ISSUE THREAD

**Issue by marscep456** [AlexeyAB/darknet#6624]
If you have an issue with training - no-detections / Nan avg-loss / low accuracy:
    * read FAQ: https://github.com/AlexeyAB/darknet/wiki/FAQ---frequently-asked-questions
    * what command do you use?darknet.exe classifier train E:/01.work/data/JIEGAN/classify_rename/data_withother/config/classify.data E:/01.work/data/JIEGAN/classify_rename/data_withother/config/csdarknet53-omega.cfg
    * what dataset do you use?   own
    * what Loss and mAP did you get? 
    * show chart.png with Loss and mAP    
    * check your dataset - run training with flag `-show_imgs` i.e. `./darknet detector train ... -show_imgs` and look at the `aug_...jpg` images, do you see correct truth bounded boxes?
    * rename your cfg-file to txt-file and drag-n-drop (attach) to your message here
    * show content of generated files `bad.list` and `bad_label.list` if they exist
    * Read `How to train (to detect your custom objects)` and `How to improve object detection` in the Readme: https://github.com/AlexeyAB/darknet/blob/master/README.md
    * show such screenshot with info
```
./darknet detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights data/dog.jpg
 CUDA-version: 10000 (10000), cuDNN: 7.4.2, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.2.0
 0 : compute_capability = 750, cudnn_half = 1, GPU: GeForce RTX 2070
net.optimized_memory = 0
mini_batch = 1, batch = 8, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
```
I used the csdarknet53-omega. cfg file to train the classification network(not detection) on my own dataset. When using label_smooth_eps =0.1, the loss increases gradually, no matter how much learning rate I set. But when not use label_smooth_eps=0.1, the loss gradually converges.I mean, why is that?
By the way, I have five categories with an average of 1,800 pictures in each category. How many training times and batch sizes should be set?

**Comment by tylertroy** [comment]
I too have found this issue. Can anyone else confirm this behavior. If so what is the purpose of `label_smooth_eps`?

**Comment by akashAD98** [comment]
Class label smoothing is not an image manipulation technique, but rather an intuitive change to class labeling. Generally, the correct classification for a bounding box is represented as a one hot vector of classes [0,0,0,1,0,0, ...] and the loss function is calculated based on this representation. However, when a model becomes overly sure with a prediction close to 1.0, it is often wrong, overfit, and over looking the complexities of other predictions in some way. Following this intuition, it is more reasonable to encode the class label representation to value that uncertainty to some degree. Naturally, the authors choose 0.9, so [0,0,0,0.9, 0....] to represent the correct class
